index.md

```diff
@@ -33,6 +33,6 @@ ranging from laptops to high-performance computers.
 FEniCS is a [NumFOCUS](https://www.numfocus.org/) fiscally supported
 project. If you like FEniCS and want to support our mission to produce
 the best possible platform for open-source computing, consider making [a
-donation](https://numfocus.salsalabs.org/donate-to-fenics/index.html) to our project.
```

Comment:
Simply change this to
```
donation](https://numfocus.org/donate-to-fenics) to our project.
```

Reply:
oops must've gotten mixed up in my copy pastes.... thanks